src/openms/source/ANALYSIS/OPENSWATH/MRMAssay.cpp

```diff
@@ -405,245 +563,282 @@ namespace OpenMS
     exp.setProteins(proteins);
   }
 
-  void MRMAssay::uisTransitions(OpenMS::TargetedExperiment& exp, std::vector<String> fragment_types, std::vector<size_t> fragment_charges, bool enable_losses, bool enable_uis_scoring, bool enable_site_scoring, double mz_threshold, std::vector<std::pair<double, double> > swathes)
+  void MRMAssay::uisTransitions(OpenMS::TargetedExperiment& exp, std::vector<String> fragment_types, std::vector<size_t> fragment_charges, bool enable_specific_losses, bool enable_unspecific_losses, double mz_threshold, std::vector<std::pair<double, double> > swathes, int round_decPow, size_t max_num_alternative_localizations, int shuffle_seed)
```

Comment:
this functions seems to get longer and longer. now reaching 276 lines. any chance we can split it up?